src/recipes/ios-promo/config.js

```diff
@@ -21,13 +21,13 @@ let thankyou = "We hope that you enjoy Firefox on your mobile device!";
 let url      = "https://www.mozilla.org/en-US/firefox/mobile-download/";
 let button   = "Get it now";
 let branches = [
-  {
+  /*{
```

Comment:
so this is the fail branch?  put in a comment saying so.   // lowest performing branch